returnn/datasets/util/feature_extraction.py

```diff
@@ -0,0 +1,452 @@
+"""
+ExtractAudioFeatures class and related helpers
+"""
+import numpy
```

Comment:
I would add an empty line before the first comment, after the module docstring, for consistency.